src/cli/list.rs

```diff
@@ -372,8 +395,8 @@ fn create_package_to_output<'a, 'b>(
     let version = package.version().into_owned();
 
     let kind = match package {
-        PackageExt::Conda(_) => "conda".to_string(),
-        PackageExt::PyPI(_, _) => "pypi".to_string(),
+        PackageExt::Conda(_) => KindPackage::Conda,
```

Comment:
could be a `From` trait, but not sure here

Reply:
Done